client/src/App.js

```diff
@@ -56,16 +64,45 @@ function App() {
     }
   }
 
+  function addGroup(groupToAdd) {
+    setAvailableGroups([...availableGroups, groupToAdd]);
+  }
+
+  function deleteGroup(groupToDelete) {
```

Comment:
The delete function does sucessfully delete the group however the the group only disappears on reload on the home screen at the moment. This is related to how you render the groups in the home component (I added a comment there as well)